packages/react-icons/src/createIcon.tsx

```diff
@@ -36,6 +36,8 @@ export interface SVGIconProps extends Omit<React.HTMLProps<SVGElement>, 'size' |
   size?: IconSize | keyof typeof IconSize;
   title?: string;
   noVerticalAlign?: boolean;
+  isSymbol?: boolean;
+  isSymbolRef?: boolean;
```

Comment:
As mentioned, this naming could be misleading since this might imply that it is a property passed to a `ref` attribute somewhere in JSX.

A possible rename here and descriptions for both of these props would come in handy.